content/zh-cn/case-studies/daocloud/index.html

```diff
@@ -0,0 +1,264 @@
+---
+title: DaoCloud 案例分析
+linkTitle: DaoCloud
+case_study_styles: true
+cid: caseStudies
+logo: daocloud_featured_logo.svg
+
+css: /css/style_daocloud.css
+new_case_study_styles: true
+heading_background: /images/case-studies/daocloud/banner1.jpg
+heading_title_logo: /images/daocloud-light.svg
+subheading: >
+  为数字世界寻找全局最优解
+case_study_details:
+  - Company: DaoCloud
+  - Location: 中国上海
+  - Industry: 云原生
+---
+
+<!--
+title: DaoCloud Case Study
+linkTitle: DaoCloud
+case_study_styles: true
+cid: caseStudies
+logo: daocloud_featured_logo.svg
+
+css: /css/style_daocloud.css
+new_case_study_styles: true
+heading_background: /images/case-studies/daocloud/banner1.jpg
+heading_title_logo: /images/daocloud-light.svg
+subheading: >
+  Seek Global Optimal Solutions for Digital World
+case_study_details:
+  - Company: DaoCloud
+  - Location: Shanghai, China
+  - Industry: Cloud Native
+-->
+
+<!--
+<h2>Challenges</h2>
+-->
+<h2>挑战</h2>
+
+<!--
+<p><a href="https://www.daocloud.io/en/">DaoCloud</a>, founded in 2014, is an innovation leader in the field of cloud native. It boasts independent intellectual property rights of core technologies for crafting an open cloud platform to empower the digital transformation of enterprises.</p>
+-->
+<p>作为云原生领域的创新领导者，<a href="https://www.daocloud.io/en/">DaoCloud</a> 成立于 2014 年底，拥有自主知识产权的核心技术，致力于打造开放的云操作系统，为企业数字化转型赋能。</p>
```

Comment:
```suggestion
<p>作为云原生领域的创新领导者，<a href="https://www.daocloud.io/">DaoCloud</a> 成立于 2014 年底，
拥有自主知识产权的核心技术，致力于打造开放的云操作系统，为企业数字化转型赋能。</p>
```
这种 `<p></p>` 应该也可以参照 [k8s 风格指南](https://kubernetes.io/zh-cn/docs/contribute/localization_zh/#layout-format)折行吧

Reply:
I will try it later

Reply:
The preview seems good!